tests/client/test_pool.py

```
@@ -0,0 +1,153 @@
+    path = tmp_path / "auth_token.json"
+    with (
+        patch.dict(os.environ, {"TRAKT_AUTH_TOKEN_PATH": str(path)}),
+        patch("client.auth.client.AUTH_TOKEN_FILE", str(path)),
+    ):
+        yield path
+
+
+def _write_token(path: Path, access_token: str) -> None:
+    token = TraktAuthToken(
+        access_token=access_token,
+        refresh_token="refresh",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "refresh_token"

(S106)